mongoexport/mongoexport.go

```diff
@@ -101,26 +90,26 @@ func New(opts Options) (*MongoExport, error) {
 
 	err := exporter.validateSettings()
 	if err != nil {
-		return nil, SetupError{
-			Err:  err,
-			Code: util.ExitBadOptions,
+		return nil, util.SetupError{
+			Err:     err,
+			Message: "try 'mongoexport --help' for more information",
```

Comment:
If this message is the same varying only in the name of the program, then we should capture the program name, not the message and generate the message on demand.

Reply:
fixed